.github/workflows/cherryPick.yml

```diff
@@ -43,12 +43,19 @@ jobs:
         with:
           GPG_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
 
+      - name: Get previous app version
+        id: getPreviousVersion
+        uses: Expensify/App/.github/actions/javascript/getPreviousVersion@main
+        with:
+          SEMVER_LEVEL: 'PATCH'
+
+      - name: Fetch history of relevant refs
+        run: |
+          git fetch origin main staging --no-tags --shallow-exclude ${{ steps.getPreviousVersion.outputs.PREVIOUS_VERSION }}
```

Comment:
This works because all the commits we care about are on main (and we also need history on staging to auto-resolve conflicts in the git cherry-pick command)